api/src/main/java/org/pmiops/workbench/captcha/CaptchaVerificationService.java

```diff
@@ -0,0 +1,72 @@
+package org.pmiops.workbench.captcha;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import javax.inject.Provider;
+import org.pmiops.workbench.captcha.api.CaptchaApi;
+import org.pmiops.workbench.captcha.model.CaptchaVerificationResponse;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.google.CloudStorageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/** Service to verify Captcha */
+@Service
+public class CaptchaVerificationService {
```

Comment:
This looks more like a ServiceImpl than a service - please pull out the Service interface.